### ISSUE TITLE
 '>=' not supported between instances of 'list' and 'float'

### ISSUE THREAD

**Issue by serious-gist** [Shaddyjr/bee-image-classifier#1]
while % Using testing set to assess accuracy%
getting an error 
![Capture](https://user-images.githubusercontent.com/45976700/107297851-811f6100-6a74-11eb-8939-ac26d7a28d76.PNG)
 '>=' not supported between instances of 'list' and 'float'

**Comment by Shaddyjr** [comment]
Interesting. I'm not sure how to recreate what you're seeing, but I will note this project is a bit dated (using `keras` instead of `tensorflow.keras`). I wasn't able to rerun the code without other errors popping up, so I ended up upgrading to using `tensorflow.keras` and rerunning things locally. Here are my reworked imports:

```
...
from skimage.transform import rescale, resize, rotate
from skimage.color import rgb2gray
from sklearn.metrics import confusion_matrix, auc, accuracy_score
from sklearn.model_selection import train_test_split

from tensorflow.keras.models import Sequential
from tensorflow.keras.layers import Dense, Dropout, Activation, Flatten
from tensorflow.keras.layers import Conv2D, MaxPooling2D
from tensorflow.keras.preprocessing.image import ImageDataGenerator
from tensorflow.keras.optimizers import Adam, RMSprop, SGD
from tensorflow.keras import callbacks
from tensorflow.keras.models import load_model
from tensorflow.keras import models

from keras.utils import np_utils
...
```

Also, I wouldn't recommend loading in the pickled model, as it is the outdated `keras` model object. You'd have to train the model (I'd suggest using epochs = 1 just to test things are working)

With these changes I was able to get the code back up and running (see image below).
![Capture](https://user-images.githubusercontent.com/13326591/107332124-bf6f4d00-6a81-11eb-9e81-3219fbee4c7c.PNG)

If you're still seeing the issue, please provide some detail on how I might recreate the error. Otherwise, if this fixes things, I'll be sure to update the repo (it could use an update, anyway 😅).

**Comment by Shaddyjr** [comment]
I've updated the repo, so closing this issue for now.
